package.json

```diff
@@ -79,34 +79,35 @@
     ]
   },
   "dependencies": {
-    "@angular/animations": "^8.0.0",
-    "@angular/bazel": "^8.0.0",
-    "@angular/cdk": "^8.0.0",
-    "@angular/cli": "^8.0.0",
-    "@angular/common": "^8.0.0",
-    "@angular/compiler": "^8.0.0",
-    "@angular/compiler-cli": "^8.0.0",
-    "@angular/core": "^8.0.0",
-    "@angular/forms": "^8.0.0",
-    "@angular/material": "^8.0.0",
-    "@angular/platform-browser": "^8.0.0",
-    "@angular/platform-browser-dynamic": "^8.0.0",
-    "@angular/platform-server": "^8.0.0",
-    "@angular/router": "^8.0.0",
+    "@angular/animations": "^9.0.0-rc.0",
+    "@angular/bazel": "^9.0.0-rc.0",
+    "@angular/cdk": "^9.0.0-rc.0",
+    "@angular/cli": "^9.0.0-rc.0",
+    "@angular/common": "^9.0.0-rc.0",
+    "@angular/compiler": "^9.0.0-rc.0",
+    "@angular/compiler-cli": "^9.0.0-rc.0",
+    "@angular/core": "^9.0.0-rc.0",
+    "@angular/forms": "^9.0.0-rc.0",
+    "@angular/material": "^9.0.0-rc.0",
+    "@angular/platform-browser": "^9.0.0-rc.0",
+    "@angular/platform-browser-dynamic": "^9.0.0-rc.0",
+    "@angular/platform-server": "^9.0.0-rc.0",
+    "@angular/router": "^9.0.0-rc.0",
     "@applitools/eyes-cypress": "^3.4.12",
-    "@bazel/buildifier": "^0.22.0",
+    "@bazel/buildifier": "^0.29.0",
     "core-js": "^2.5.4",
     "hammerjs": "^2.0.8",
     "opencollective": "^1.0.3",
-    "rxjs": "^6.4.0",
-    "zone.js": "^0.9.1"
+    "rxjs": "^6.5.3",
+    "tslib": "^1.10.0",
+    "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.800.0",
-    "@bazel/bazel": "0.26.0",
+    "@angular-devkit/build-angular": "~0.900.0-rc.0",
+    "@bazel/bazel": "1.0.0",
```

Comment:
Perhaps I took it too far here to also upgrade bazel, I can revert/create a separate PR for it.
Having some troubles to get it working during CI tho, and I'm not able to reproduce it locally.

Reply:
Probably need to bump the cache key here for CI to 12 https://github.com/ngrx/platform/blob/master/.circleci/config.yml#L13

Reply:
Should we modify the cache key from the `master` branch?

I tried bumping the version of this branch, but with the same result.
Seems related to https://github.com/bazelbuild/rules_nodejs/issues/1027, I already tried re-running multiple times.